docs/en/ingest-management/fleet/add-fleet-server-kubernetes.asciidoc

```diff
@@ -0,0 +1,392 @@
+== Default port assignments
+
+When {es} or {fleet-server} are deployed, components communicate over well-defined, pre-allocated ports.
+You may need to allow access to these ports. Refer to the following table for default port assignments:
+
+|===
+| Component communication | Default port
+| {agent} → {fleet-server} | 8220
+| {fleet-server} → {es} | 9200
+| {fleet-server} → {kib} (optional, for {fleet} setup) | 5601
+| {agent} → {es} | 9200
+| {agent} → Logstash | 5044
+| {agent} → {kib} (optional, for {fleet} setup) | 5601
+|===
+
+In Kubernetes environments, you can adapt these ports without modifying the listening ports of the {fleet-server} or other applications, as traffic is managed by Kubernetes `Services`. This guide includes an example where {agent}s connect to the {fleet-server} through port `443` instead of the default `8220`.
+
+[discrete]
+[[add-fleet-server-kubernetes-add-server]]
+= Add {fleet-server}
+
+A {fleet-server} is an {agent} that is enrolled in a {fleet-server} policy. The policy configures the agent to operate in a special mode to serve as a Fleet Server in your deployment.
+
+[discrete]
+[[add-fleet-server-kubernetes-preparations]]
+== Fleet preparations
+
+[TIP]
+====
+If you already have a Fleet policy with the {fleet-server} integration, you know its ID, and you know how to generate an {ref}/service-tokens-command.html[{es} service token] for the {fleet-server}, skip directly to <<add-fleet-server-kubernetes-install>>.
+
+Also note that the `service token` required by the {fleet-server} is different from the `enrollment tokens` used by {agent}s to enroll to Fleet.
+====
+
+. In {kib}, open *Fleet → Settings* and ensure the *Elasticsearch output* that will be used by the {fleet-server} policy is correctly configured, paying special attention to:
++
+** The *hosts* field includes a valid URL that will be reachable by the {fleet-server} Pod.
+** If your {es} cluster uses certificates signed by private or intermediate CAs not publicly trusted, ensure to add the trust information in the *Elasticsearch CA trusted fingerprint* field or in the *advanced configuration* section through `ssl.certificate_authorities` setting. For more details refer to https://elastic.co/guide/en/fleet/current/secure-connections.html#_encrypt_traffic_between_elastic_agents_fleet_server_and_elasticsearch[Secure Connections] documentation.
++
+[IMPORTANT]
+====
+This validation step is critical. The {es} host URL and CA information has to be added *in both the {es} output and the environment variables* provided to the {fleet-server} in a later step. It's a common mistake to ignore the output settings believing that the environment variables will prevail, when the environment variables are only used during the bootstrap of the {fleet-server}.
+
+Consider the creation of a dedicated *{es} output* for the {fleet-server} if the URL needed by the {fleet-server} to reach {es} differs from the URL needed by other clients.
```

Comment:
```suggestion
If the URL that {fleet-server} will use to access {es} is different from the {es} URL used by other clients, you may want to create a dedicated *{es} output* for {fleet-server}.
```
I think I'd reverse the order to make it a bit more clear, but the original is okay too.